controllers/articles-controller.js

```diff
@@ -1,18 +1,13 @@
-const e = require('express')
 const { getArticlesById } = require('../models/articles')
 
 exports.findArticlesById = (req,res, next) => {
     const {article_id} = req.params
     if(!/[1-9]/.test(article_id)){
```

Comment:
I like what you're doing here! You're more than welcome to at least TRY to query the database and let it generate an error for you - to then be caught, passed to `next` and sorted out by an error handler